cardano-db/src/Cardano/Db/Migration.hs

```diff
@@ -238,7 +239,10 @@ createMigration source (MigrationDir migdir) = do
     create :: ReaderT SqlBackend (NoLoggingT IO) (Maybe (MigrationVersion, Text))
     create = do
       ver <- getSchemaVersion
-      statements <- getMigration migrateCardanoDb
+      -- here is the place to combine any "core" schemas to the base schema
+      statementsBase <- getMigration migrateBaseCardanoDb
+      statementsTxOut <- getMigration migrateCoreTxOutCardanoDb
```

Comment:
I thought that since tx_out core migration has access to the base migration, only the former needs to be used, but apparently I was wrong?

Also, could we have a flag here, to allow creating the schema for the variant?

Reply:
I tried with just `statementsBase` then just `statementsTxOut` but they both failed so I put them together. 

I added the option when running the migration but I'll add one for creating migration too 👍